canopeum_backend/canopeum_backend/views.py

```diff
@@ -347,14 +348,16 @@ def patch(self, request: Request, siteId):
         serializer = SiteSerializer(site, data=request.data, partial=True)
         if serializer.is_valid():
             site = serializer.save(
-                image=image,
+                image=image if request.data.get("image") is not None else site.image,  # pyright: ignore[reportPossiblyUnboundVariable]
```

Comment:
You should instead first assign `image = site.image` in the else path of `if request.data.get("image") is not None:`

Note that we should probably eventually differenciate between "do nothing" (no image param) and "remove image" (image param set to None ?)

```suggestion
                image=image,
```